README.md

```diff
@@ -297,6 +297,7 @@ The OpenTelemetry Operator *might* work on versions outside of the given range,
 
 | OpenTelemetry Operator | Kubernetes           | Cert-Manager         |
 |------------------------|----------------------|----------------------|
+| v0.52.0                | v1.19 to v1.23       | v1                   |
```

Comment:
Is the cert-manager v1 here because we updated to 1.18?

Reply:
Yes.